src/lib/rest/RestService.cpp

```diff
@@ -149,7 +149,7 @@ std::string payloadParse
 *
 * tenantCheck - 
 */
-static std::string tenantCheck(const std::string& tenant)
+extern std::string tenantCheck(const std::string& tenant)
```

Comment:
If our common style in other parts is not including extern in this case (as I understand it is) I'll suggest to do it in the same way also here.